package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "movie-website",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
[Optional] Kindly add a description of the project.

Reply:
Thanks for your suggestion :+1: